app/code/Magento/MediaGalleryUi/etc/adminhtml/menu.xml

```diff
@@ -7,7 +7,21 @@
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Backend:etc/menu.xsd">
     <menu>
-        <add id="Magento_MediaGalleryUi::media" title="Media" translate="title" module="Magento_MediaGalleryUi" sortOrder="15" parent="Magento_Backend::content" resource="Magento_Cms::media_gallery" dependsOnConfig="system/media_gallery/enabled"/>
-        <add id="Magento_MediaGalleryUi::media_gallery" title="Media Gallery" translate="title" module="Magento_MediaGalleryUi" sortOrder="0" parent="Magento_MediaGalleryUi::media" action="media_gallery/media/index" resource="Magento_Cms::media_gallery"/>
+        <add id="Magento_MediaGalleryUi::media"
+             title="Media"
+             translate="title"
+             module="Magento_MediaGalleryUi"
+             sortOrder="15"
+             parent="Magento_Backend::content"
+             resource="Magento_Cms::media_gallery"
+             dependsOnConfig="system/media_gallery/enabled"/>
```

Comment:
@sivaschenko ...
Do we need to display this menu in old media gallery?
Thanks

Reply:
Only Sores configuration representation should be changed in scope of the issue

Reply:
But I didn't understood if `Enable Old Media Gallery` is set to Yes then the menu will be visible